ts/editor/Fields.svelte

```diff
@@ -10,7 +10,6 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     .fields {
         display: grid;
         grid-auto-rows: min-content;
-        grid-gap: 6px;
```

Comment:
You don't need to replace grid-gap.

`grid-gap` is well supported, the only thing not supported is `flex-gap`, or in other words, using `gap` with `display: flex`